packages/verkle/src/node/baseVerkleNode.ts

```diff
@@ -2,24 +2,22 @@ import { RLP } from '@ethereumjs/rlp'
 
 import { type VerkleNodeInterface, type VerkleNodeOptions, type VerkleNodeType } from './types.js'
 
+import type { VerkleCrypto } from 'verkle-cryptography-wasm'
```

Comment:
This should now use the import from `verkle` in `Util`, just opened a related issue here https://github.com/ethereumjs/verkle-cryptography-wasm/issues/54

I would go as far and argue that we should *also* remove the `verkle-cryptography-wasm` dependency from this package as well and dependency inject, and so to have this wider strategic goal over the head that this should be replaceable and also here optimally a JS implementation being used.

We then also automatically continue to evolve a generic interface around this and keep thinking about what's needed and what not (I e.g. still like the idea that we take the more hashing related parts out *directly* and move over to the pedersen/poseidon hash usage from Pauls libraries https://github.com/paulmillr/scure-starknet (if applicable) and with this slowly shrink down the surface of what we need to compile over into the WASM stuff.

Feels to me that there is still too much in. 

Practical note: if depency injection feels a bit impractical for now for testing and the like we could also decide to do at a later stage.

Reply:
Ah, but the main argument for direct dependency injection (so, this would mean: having a mandatory `crypto` constructor argument) is actually that we avoid having the same bundle problems again once/if we have Verkle added somewhere as a dependency.